.pre-commit-config.yaml

```diff
@@ -9,6 +9,7 @@ repos:
         entry: civicactions/guidebook-lint:latest remark.sh
         always_run: true
         pass_filenames: false
+        verbose: true
```

Comment:
@grugnog any issue with enabling verbose for these checks that you can see?

Reply:
Oh..I updated that because I wanted to see the suggestions on commit.  They were only showing if there was also an error.  I can remove it, but it was helpful for me.

Reply:
That is fair. I am just not sure if our checks would care so I was curious. We can keep it and try it out.